src/pretalx/agenda/templates/agenda/schedule.html

```diff
@@ -110,4 +110,12 @@ <h3 id="{{ day.start.date|date:"c" }}">
         </div>
     {% endfor %}
 </div>
+
+
+<script>
```

Comment:
If at all possible, please do not use inline scripts, as it makes it harder to write a good CSP. I am aware that we've got a few already, though ;)